example/libhdbpp-example/CHANGELOG.md

```diff
@@ -0,0 +1,91 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Changed
+
+- Removed public headers as the only needed one is AbstractDB from libhdbpp.
```

Comment:
CHANGELOG.md looks copy-pasted from libhdbpp-timescale, it should be cleaned

Reply:
oupsie… You caught me, thanks !